cmd/kpod/inspect.go

```diff
@@ -0,0 +1,135 @@
+package main
+
+import (
+	"encoding/json"
+	"fmt"
+	"os"
+	"text/template"
+
+	"github.com/pkg/errors"
+	"github.com/urfave/cli"
+)
+
+const (
+	defaultFormat = `Container: {{.Container}}
+ID: {{.ContainerID}}
+`
+	inspectTypeContainer = "container"
+	inspectTypeImage     = "image"
+	inspectAll           = "all"
+)
+
+var (
+	inspectFlags = []cli.Flag{
+		cli.StringFlag{
+			Name:  "type, t",
+			Usage: "look at the item of the specified `type` (container or image) and name",
+		},
+		cli.StringFlag{
+			Name:  "format, f",
+			Usage: "use `format` as a Go template to format the output",
+		},
+		cli.BoolFlag{
+			Name:  "size",
+			Usage: "display total file size if the type is container",
+		},
+	}
+	inspectDescription = "Displays a build container's or built image's configuration."
+	inspectCommand     = cli.Command{
+		Name:        "inspect",
+		Usage:       "Displays the configuration of a container or image",
+		Description: inspectDescription,
+		Flags:       inspectFlags,
+		Action:      inspectCmd,
+		ArgsUsage:   "CONTAINER-OR-IMAGE",
+	}
+)
+
+func inspectCmd(c *cli.Context) error {
+	var data *containerImageData
+	args := c.Args()
+	if len(args) == 0 {
+		return errors.Errorf("container or image name must be specified")
+	}
+	if len(args) > 1 {
+		return errors.Errorf("too many arguments specified")
+	}
+
+	itemType := inspectAll
+	if c.IsSet("type") {
+		itemType = c.String("type")
+	}
+	switch itemType {
+	case inspectTypeContainer:
+	case inspectTypeImage:
+	case inspectAll:
+	default:
+		return errors.Errorf("the only recognized types are %q, %q, and %q", inspectTypeContainer, inspectTypeImage, inspectAll)
+	}
+
+	format := defaultFormat
+	if c.IsSet("format") {
+		if c.String("format") != "" {
+			format = c.String("format")
+		}
+	}
+	t := template.Must(template.New("format").Parse(format))
+
+	name := args[0]
+
+	store, err := getStore(c)
+	if err != nil {
+		return err
+	}
+
+	size := false
+	if c.IsSet("size") {
+		size = c.Bool("size")
```

Comment:
Is this enough.  If it is not set won't 

size := c.Bool("size") 
end up with size == false.